cli/remotebazel/remotebazel.go

```diff
@@ -167,27 +169,67 @@ func determineRemote(repo *git.Repository) (*git.Remote, error) {
 }
 
 func determineDefaultBranch(repo *git.Repository) (string, error) {
-	branches, err := repo.Branches()
+	f := sync.OnceValues(func() (string, error) {
```

Comment:
this `OnceValues` won't work as intended, because it's being redefined on each invocation of `determineDefaultBranch`.

Reply:
I removed the OnceValues usage entirely.

I guess it's not "safe" to assume `repo` will not change between func calls? 🤔

Reply:
It probably is safe to assume `repo` won't change, because it refers to the local repo state. Seems unlikely you'd change the default branch while simultaneously running remote bazel on the same machine

Looks like we call `determineDefaultBranch` twice - you could just save the values to vars and pass them into the functions that use them so we don't have to do the potentially expensive remote calls twice